server/api/sample_sizes.py

```diff
@@ -64,18 +65,26 @@ def sample_sizes_for_contest(contest: Contest):
             assert election.audit_type == AuditType.BALLOT_COMPARISON
 
             set_contest_metadata_from_cvrs(contest)
-            # TODO compute sample_results
-            ballot_comparison_sample_results: Dict[str, Union[int, float]] = {
-                "sample_size": 0,
-                "1-under": 0,
-                "1-over": 0,
-                "2-under": 0,
-                "2-over": 0,
+            contest_for_sampler = sampler_contest.from_db_contest(contest)
+
+            discrepancies = supersimple.compute_discrepancies(
+                contest_for_sampler,
+                rounds.cvrs_for_contest(contest),
+                rounds.sampled_ballot_interpretations_to_cvrs(contest),
+            )
+            discrepancy_groups = group_by(
+                discrepancies.values(), lambda d: d["counted_as"]
+            )
+            discrepancy_counts = {
+                "sample_size": len(discrepancies),  # TODO is this the right value?
```

Comment:
This isn’t the right value. Can you get the sample size from earlier in this function?

Reply:
Or I could reconfigure the discrepancy function to work this way, whichever is easier.

Reply:
what is the right number? is it the total number of samples drawn in the audit thus far? if so, probably need to query the db for it - not a big deal

Reply:
Yeah, the total number of samples drawn so far

Reply:
ok how does it look now? c2d0026

Reply:
Perfect :)